README.md

```diff
@@ -5,33 +5,23 @@ ffagc
 
 ## Initial Environment Setup
 
-build / install stuff I dunno who even uses rails any more:
+Set up the project with:
 
-    rake
-    bundle update
-    bundle install
+```sh
+  bundle install
+  bundle exec rake db:migrate RAILS_ENV=development
+  bundle exec rails server
+```
 
-You'll need to create config/secrets.yml with the form:
-
-    development:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    test:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    production:
-      secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-
-Use 'rake secret' to generate the hex.  Don't check this file into source control!
+If you are running this project in production you'll need to set environment variables for secrets that include `ENV` in `config/secrets.yml`.
 
 ## Email Setup
 
-If you plan to send email, you'll need to create config/initializers/smtp_secret.rb
-with:
-
-    ENV['PASSWORD']="your smtp password"
+If using SMTP set the required ENV variables or fill in (and do not commit) `.env` like so:
 
-and edit config/environments/*.rb to include your smtp server information.
```

Comment:
Maybe reword for clarity.  The audience for this readme is not necessarily going to be familiar with Rails:

"If you're using SMTP to send emails, set the required ENV variables in config/secrets.yml or create a hidden file /[your repo]/.env (and do not add it to the repository) that overrides the values you want to set like so:

SMTP_PASSWORD='this_is_not_the_password'
# ... more values as you need
"